sycl/doc/GetStartedGuide.md

```diff
@@ -197,6 +193,10 @@ CL_CONFIG_DEVICES = fpga-emu
 2) Create ICD file pointing to the new runtime
 
     ```bash
+    # OpenCL FPGA emulation RT
+    echo  /opt/intel/oclfpgaemu_<fpga_version>/x64/libintelocl_emu.so >
```

Comment:
NIT: this command (and maybe some others) will require root access, could be mentioned somewhere